coffeecard/CoffeeCard.Common/Configuration/SmtpSettings.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+using NetEscapades.Configuration.Validation;
+
+namespace CoffeeCard.Common.Configuration
+{
+    public class SmtpSettings : IValidatable
```

Comment:
We should probably make it optional, as I am also realizing I would need to update the Bicep for deploying the appsettings. But that seems overkill since it is not used. Do you know how I could make it optional or only required in local development? I guess I could change the validator, or just not have it be validated ?